packages/components/data-resource/s3-endpoint-type.tsx

```diff
@@ -121,10 +125,14 @@ export const S3EndPointType: React.FC<S3EndpointTypeProps> = (props) => {
           <InputGroup>
             <ResourceDropdown
               className="nb-endpoints-form-entry__dropdown nb-endpoints-form-entry__dropdown--full-width"
-              resource={secretResource}
+              resource={{
+                ...secretResourceWithoutNamespace,
+                namespaced: false,
+              }}
               resourceModel={SecretModel}
               onSelect={getSecrets}
               initialSelection={getInitialSelection}
```

Comment:
test this once for dedicated-user, whether it is working or not...

Reply:
getting 403 - working on it

![image](https://user-images.githubusercontent.com/22158580/174973034-58245845-11f1-48cf-a161-085b81ce3be8.png)